### ISSUE TITLE
Support for Mutiny

### ISSUE THREAD

**Issue by Plawn** [rpuch/pulsar-reactive-client#1]
In the readme, Mutiny is written to be in consideration for this library, is it still the case ?

**Comment by rpuch** [comment]
Yes, it is, but I had no time to invest in this yet. Contributions are welcome ;)
